spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -67,4 +67,8 @@ def enable_arkivo_api
   def relax_routing_constraint
     gsub_file 'config/initializers/arkivo_constraint.rb', 'false', 'true'
   end
+
+  def override_devise_registrations_controller
```

Comment:
See also: https://github.com/samvera-labs/hyrax/pull/1151#issue-233985835

This could arguably move into Hyrax's install generator to "automatically" activate this flipper in new installs. (And should be documented in the 2.0.0 release notes, so folks who are upgrading know to make this change in their installations to make this flipper work.) Would like in put on that.